django_project/purchases/admin.py

```diff
@@ -1,3 +1,24 @@
 from django.contrib import admin
 
-# Register your models here.
+from purchases.models import Purchase
+
+
+@admin.register(Purchase)
+class PurchaseAdmin(admin.ModelAdmin):
+    list_display = ('id', 'get_user_id', 'get_book_id', 'purchase_date')
```

Comment:
nit: I think it is better to show 'user', 'book'. it shows info from `__str__` of these models